Plugins/IngameDebugConsole/Scripts/InputHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using UnityEngine;
+
+#if ENABLE_INPUT_SYSTEM
+using UnityEngine.InputSystem;
+
+namespace IngameDebugConsole
+{
+    public static class InputHelper
+    {
+        public static bool GetKeyDown(KeyCode key)
+        {
+            bool ok = Enum.TryParse(key.ToString(), out Key newKey);
```

Comment:
This line must be optimized so that either Enum.TryParse is never called or the result is cached in a Dictionary so that TryParse is called only once per key.

Reply:
Good Idea, i gonna check to use a Dictionnary at this place.

Reply:
I have correct this by mapping all keys existing in both Input Systems in a big Dictionary.